### PR TITLE
Add new maintainers to the plugin repo

### PR DESCRIPTION
**Pull request by bhavyalatha26**
# Link to GitHub repository

# When modifying release permission

List the GitHub usernames of the users who should have commit permissions below:
- `@bhavyalatha26`
- `@nazrul-cloudaeye`
- `@PiyushKumar11`

Checklist:

```[tasklist]
### Release permission checklist (for submitters)
- [x] The usernames of the users added to the "developers" section in the .yml file are the same the users use to log in to [accounts.jenkins.io](https://accounts.jenkins.io/).
- [x] All users added have logged in to [Artifactory](https://repo.jenkins-ci.org/) and [Jira](https://issues.jenkins.io/) once.
- [x] I have mentioned an [existing team member](https://github.com/orgs/jenkinsci/teams) of the plugin or component team to approve this request.
```

```[tasklist]
### Reviewer checklist
- [ ] Check that the `$pluginId Developers` team has `Admin` permissions while granting the access.
- [ ] In the case of plugin adoption, ensure that the Jenkins Jira default assignee is either removed or changed to the new maintainer.
- [ ] If security contacts are changed (this includes add/remove), ping the security officer (currently `@Wadeck`) in this pull request. If an email contact is changed, wait for approval from the security officer.
```
There are [IRC Bot commands](https://jenkins.io/projects/infrastructure/ircbot/#issue-tracker-management) for it.

### CODE REVIEW

**Comment by bhavyalatha26** [comment]
`piyush_1` logged in now, please re-trigger

**Comment by bhavyalatha26** [comment]
@timja can you please review this request and merge the PR.
TIA

**Comment by timja** [comment]
Please fill out the full PR template properly and don't remove anything:
https://github.com/jenkins-infra/repository-permissions-updater/blob/master/.github/PULL_REQUEST_TEMPLATE.md

File: permissions/plugin-cloudaeye.yml [hunk 1]
```diff
@@ -5,5 +5,9 @@ paths:
 - "io/jenkins/plugins/cloudaeye"
 developers:
 - "cloudaeye"
+- "nazrul"
+- "piyush_1"
 issues:
   - github: *GH
+cd:
+  enabled: true
```
Comment (alecharp) [inline permissions/plugin-cloudaeye.yml:13]:
```suggestion
  enabled: true

```

**Comment by alecharp** [comment]
We need the user behin the username `cloudaeye` to approve this pull request as well.

**Comment by timja** [comment]
> We need the user behin the username `cloudaeye` to approve this pull request as well.

https://github.com/jenkins-infra/repository-permissions-updater/issues/4131 this is the person.

**Comment by alecharp** [comment]
Ok then everything is good. Thank you all.

**Pull request merged**